Makefile

```diff
@@ -0,0 +1,34 @@
+CHANNEL ?= unpublished
+CHARM := sriov-cni
+
+ifndef NAMESPACE
+$(error NAMESPACE is not set)
+endif
+
+ifeq (, $(shell which lxc))
+$(error "LXC/D must be installed before proceeding")
+endif
+
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
+workdir := /$(CHARM)/$(current_dir)
+
+setup-env:
+	sudo lxc launch ubuntu:18.04 $(CHARM) && sleep 5
+	sudo lxc config set $(CHARM) security.nesting true
+	sudo lxc file push -r -p ../$(current_dir) $(CHARM)/$(CHARM)
+	sudo lxc exec $(CHARM) -- env WORKDIR=$(workdir) bash $(workdir)/script/bootstrap
+
+charm: setup-env
+	sudo lxc exec $(CHARM) -- env WORKDIR=$(workdir) bash $(workdir)/script/build
```

Comment:
The .charm file that gets built here is tucked away inside the LXD container. What if I want to extract and inspect it? I would like to be able to do that without having to dig around in the container. Can `make charm` produce a local .charm file?

Reply:
This applies to the docker image as well, which is also tucked away inside the container.